setup.py

```diff
@@ -32,6 +32,9 @@
         "fonttools>=3.1.2",
         "ufoLib>=2.1.0",
     ],
+    extras_require={
+        'pens': ["fontPens>=0.1.0", ],
```

Comment:
you could make this list multi-line if you think more could be added later (or just remove the final comma for now)